fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java

```diff
@@ -1026,6 +1029,10 @@ private boolean loadTxnCommitImpl(TLoadTxnCommitRequest request) throws UserExce
                 entity.counterRoutineLoadFinishedTotal.increase(1L);
                 entity.counterRoutineLoadBytesTotal.increase(routineAttachment.getReceivedBytes());
                 entity.counterRoutineLoadRowsTotal.increase(routineAttachment.getLoadedRows());
+                if (basicStatsMeta != null) {
+                    basicStatsMeta.increase(routineAttachment.getLoadedRows());
```

Comment:
you could update loaded rows in OlapTableTxnLogApplier.applyCommitLog, or DatabaseTransactionMgr.updateCatalogAfterCommitted  and only need do once.